tsl/test/expected/hypercore.out

```diff
@@ -138,17 +138,32 @@ WHERE time = '2022-06-01'::timestamptz;
 
 -- Create a new index on a compressed column
 CREATE INDEX ON readings (location);
--- Check that we error out on unsupported index types
+-- Run these checks as superuser to avoid permission errors. These
+-- commands should still be blocked and have reasonable error
+-- messages.
+set role :ROLE_SUPERUSER;
 \set ON_ERROR_STOP 0
+-- Check that we error out on unsupported index types
 create index on readings using brin (device);
 ERROR:  index access method "brin" not supported
 create index on readings using gin (jdata);
 ERROR:  index access method "gin" not supported
 create index on readings using magicam (device);
 ERROR:  access method "magicam" does not exist
+-- Check that we error out when trying to build index concurrently.
+create index concurrently on readings (device);
+ERROR:  hypertables do not support concurrent index creation
```

Comment:
Ahh, so hypertables don't support concurrent index builds at all? I though we had some code for that?

Reply:
Nope, this is blocked in `process_utility.c`.